assets/create-generic-asset.js

```diff
@@ -0,0 +1,128 @@
+/*
+    This example demonstrates how to create generic asset (w/o geometry).
+    The facility is using REC Sample template and new asset is Lighting Equipment.
+    
+    It uses 2-legged authentication - this requires that application is added to facility as service.
+*/
+import { createToken } from '../common/auth.js';
+import { TandemClient } from '../common/tandemClient.js';
+import { ColumnFamilies,
+    ColumnNames,
+    ElementFlags,
+    MutateActions, 
+    getDefaultModel, 
+    matchClassification} from '../common/utils.js';
+
+// update values below according to your environment
+const APS_CLIENT_ID = 'YOUR_CLIENT_ID';
+const APS_CLIENT_SECRET = 'YOUR_CLINET_SECRET';
+const FACILITY_URN = 'YOUR_FACILITY_URN';
+const CLASSIFICATION = 'Lighting Equipment';
+const ASSEMBLY_CODE = 'D5040.50'; // Lighting Fixtures
+const CATEGORY = 1120; // Lighting Fixtures
+
+async function main() {
+    // STEP 1 - obtain token to authenticate subsequent API calls
+    const token = await createToken(APS_CLIENT_ID,
+        APS_CLIENT_SECRET, 'data:read data:write');
+    const client = new TandemClient(() => {
+        return token;
+    });
+
+    // STEP 2 - get facility
+    const facilityId = FACILITY_URN;
+    const facility = await client.getFacility(facilityId);
+    const defaultModel = getDefaultModel(facilityId, facility);
```

Comment:
It is very important to emphasise importance of using a default model here to avoid potential data loss. Code comment would work.